tests/test_MOcovMFile_recognizes_classdef_syntax.m

```diff
@@ -0,0 +1,81 @@
+function test_suite = test_MOcovMFile_recognizes_classdef_syntax
+    initTestSuite;
+end
+
+function fullname = tempfile(filename, contents)
+    tempfolder = fullfile(tempdir, 'mocov_fixtures');
+    [~, ~, ~] = mkdir(tempfolder);
```

Comment:
If I am not wrong, Matlab issues a warning if `mkdir` tries to create an already existent, unless all the 3 return values are captured.

Can I opt by the `unused` suggestion?